Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml

```diff
@@ -661,7 +625,159 @@ tasks:
       {
         "position": {
           "x": 870,
-          "y": 10
+          "y": 270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '24':
+    id: '24'
+    taskid: 9b25ec71-1d64-4668-8e4c-a13e097b9350
+    type: condition
+    task:
+      id: 9b25ec71-1d64-4668-8e4c-a13e097b9350
+      version: -1
+      name: Are there related detections?
+      type: condition
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#default#':
+      - '25'
+      yes:
+      - '18'
+    separatecontext: false
+    conditions:
+    - label: yes
+      condition:
+      - - operator: isNotEqualString
+          left:
+            value:
+              simple: CrowdStrike.FoundDetections
+            iscontext: true
+          right:
+            value:
+              simple: 'False'
```

Comment:
Maybe we should add here also ignore case flag (to support both lower and upper case - even if we are talking on a key that your setting)

Reply:
As you already said
the playbooks is setting it and not the user so not relevant